e2e/handlers.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+. lib.sh
+
+setup_suite() {
+	# shellcheck disable=SC2016
+	_kubectl patch ds -n kube-system kilo -p '{"spec": {"template":{"spec":{"containers":[{"name":"kilo","args":["--hostname=$(NODE_NAME)","--create-interface=false","--kubeconfig=/etc/kubernetes/kubeconfig","--mesh-granularity=full"]}]}}}}'
```

Comment:
We can remove this patch and re-use the default cluster configuration. Unless you really want to ensure that we necessarily use full-mesh.

Reply:
Done